src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/javadoc/javadoctagcontinuationindentation/Example3.java

```diff
@@ -0,0 +1,34 @@
+/*xml
+<module name="Checker">
+  <module name="TreeWalker">
+    <module name="JavadocTagContinuationIndentation">
+      <property name="violateExecutionOnNonTightHtml" value="true"/>
+    </module>
+  </module>
+</module>
+*/
+package com.puppycrawl.tools.checkstyle.checks.javadoc.javadoctagcontinuationindentation;
+
+// xdoc section -- start
+/**
+ * <p> 'p' tag is unclosed
+ * <p> 'p' tag is closed</p>
+ */
+class Example3 {
+  /**
+   * @tag comment
+   *     Indentation spacing is 4
+   */
+  public void testMethod1(String input) {
+    // violation 9 lines above 'Unclosed HTML tag found: p'
```

Comment:
Please move this violation above javadoc comment with this problem. I found it it is ok to have such extra lines at the beginning or end, it still easy to catch diff

Reply:
Done